src/fentoimage/board.py

```diff
@@ -83,9 +95,43 @@ def _render_piece(self, x, y):
             piece_image = self.piece_drawer.render(piece)
             self.image.paste(piece_image, (rectx, recty), piece_image)
 
-    def render(self, highlighted_squares=()):
+    def _flip_fen(self, fen) -> str:
+        # get the index of the space
+        space_ind = fen.index(" ")
+        board_lines_fen = fen[0:space_ind]
+
+        # get the board lines fen split up
+        board_lines_fen = board_lines_fen.split("/")
+
+        new_fen = ""
+
+        # read board lines fen backwards for first flip
+        for i in range(len(board_lines_fen) - 1, -1, -1):
+            new_fen += board_lines_fen[i][::-1]
+
+            # if not at the end, add a slash
+            if i != 0:
+                new_fen += "/"
+
+        # add back the active colour, castling, en passant, half moves, full moves
+        new_fen = new_fen + fen[space_ind:]
+
+        return new_fen
```

Comment:
Do the available functions in the chess library not work? https://python-chess.readthedocs.io/en/latest/core.html#chess.Board.transform. I don't know tbh, I'm just asking, since I'd prefer using an existing solution rather than rolling our own.

Reply:
Thanks, good idea! I will investigate and get back